libs/UGridNET/dll/src/ContactsExtensions.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleToAttribute("UGridNET.Tests")]
+
+namespace UGridNET
+{
+    namespace Extensions
+    {
+        internal static class ContactsExtensions
+        {
+            /// <summary> Allocates unmanaged memory for all required properties of the <see cref="Contacts"/> instance. </summary>
+            /// <param name="contacts"> Instance of <see cref="Contacts"/>. </param>
+            /// <remarks> Memory allocated by this method must be freed by <see cref="Free"> </remarks>
+            public static void Allocate(this Contacts contacts)
+            {
+                try
+                {
+                    contacts.name = IntPtrHelpers.Allocate<byte>(UGrid.name_long_length);
+                    contacts.contact_name_id = IntPtrHelpers.Allocate<byte>(UGrid.name_length * contacts.num_contacts);
+                    contacts.mesh_from_name = IntPtrHelpers.Allocate<byte>(UGrid.name_long_length);
```

Comment:
UGrid standards use long_name. consider changing to long_name_length

Reply:
OK. This was allocated following dimensions used in the API unit tests.